ahl_health_apps_analysis/pipeline/streamlit_viz.py

```diff
@@ -0,0 +1,75 @@
+import streamlit as st
+import altair as alt
+import pandas as pd
+from ahl_health_apps_analysis.analysis.plotting import configure_plots
+
+date = ('2022-11-28')
+clusters = 20
+
+data = pd.read_csv(f'outputs/data/{date}-kmeans-{clusters}-refined.csv') 
```

Comment:
Now just add `outputs/data/2022-11-28-kmeans-20-refined.csv` to the repo and then this can all be run on Streamlit Cloud!